chatter.js

```diff
@@ -1,12 +1,80 @@
 Chatter = {
-    options: {
-        messageLimit: 30,
-        nickProperty: "username",
+  options: {
+    messageLimit: 30,
+    nickProperty: "username",
+  }
+};
+
+Chatter.configure = function (opts) {
+  _.extend(this.options, opts);
+};
+
+/**
+ * @summary Adds user to Chatter and defines its privileges
+ * @locus Server
+ * @param {string} userId Unique string identifying the users of your application.
+ * @param {string} userType Defines the restricitons on the new user. Can either be set to "admin" or "standard".
+ * @returns {string} userId
+ */
+Chatter.addUser = function(userId, userType) {
+   check(userId, String);
+   check(userType, Match.Maybe(String));
+  const user = Meteor.users.findOne({_id: userId});
+
+  Chatter.User.upsert({
+    userId: userId
+  }, {
+    $set: {
+      userType: userType,
+      nickname: getNickname(user)
     }
+  });
+  return Chatter.User.findOne({userId: userId})._id;
 };
 
+/**
+ * @summary Adds empty room to Chatter.
+ * @locus Server
+ * @param {string} rommName
+ * @returns {string} roomId
+ */
+Chatter.addRoom = function(roomName) {
+  check(roomName, String);
 
-Chatter.configure = function (opts) {
-    _.extend(this.options, opts);
+  return Chatter.Room.insert({name: roomName});
 };
 
+/**
+ * @summary Adds user to room.
+ * @locus Server
+ * @param {string} userId Unique string identifying user.
+ * @param {string} roomName Unique string identifying the room.
+ * @returns {string} userRoomId
+ */
+Chatter.addUserToRoom = function(userId, roomId) {
+  check(userId, String);
+  check(roomId, String);
+  const chatterUserId = Chatter.User.findOne({userId: userId})._id;
+  const room = Chatter.Room.findOne({_id: roomId});
+
+  if (room === undefined || chatterUserId === undefined) {
+    return "room or user does not exist";
+  }
+
+  return Chatter.UserRoom.upsert({
+    userId: chatterUserId,
```

Comment:
Good point. This has been on my list for while. I have changed all variables to chatterUserId, and chatterUser. Will that suffice or should I also rename the attributes of the chatter models, so that when creating new instances in becomes this:

return Chatter.User.upsert({
+    chatterUserId: chatterUserId,
+    roomId: roomId
});

rather than this:
return Chatter.User.upsert({
+    userId: chatterUserId,
+    roomId: roomId
});